tests/fixtures/here_travel_time/routing_error_invalid_credentials.json

```diff
@@ -1,15 +1,4 @@
 {
-    "_type": "ns2:RoutingServiceErrorType",
-    "type": "PermissionError",
-    "subtype": "InvalidCredentials",
-    "details": "This is not a valid app_id and app_code pair. Please verify that the values are not swapped between the app_id and app_code and the values provisioned by HERE (either by your customer representative or via http://developer.here.com/myapps) were copied correctly into the request.",
-    "metaInfo": {
-        "timestamp": "2019-07-10T09:43:14Z",
-        "mapVersion": "8.30.98.152",
-        "moduleVersion": "7.2.201927-4307",
-        "interfaceVersion": "2.6.64",
-        "availableMapVersion": [
-            "8.30.98.152"
-        ]
-    }
-}
\ No newline at end of file
+    "error": "Unauthorized",
```

Comment:
unrelated commit

Reply:
This is related. We can only have one version of the herepy dependency.
As `here_travel_time` and `here_weather` are using the same dependency I had to update them.

`here_travel_time` does not mock the lib for tests but mocks the api response. This is why I had to adjust the fixture for the new lib version as well